web/helpers/project_helper.rb

```diff
@@ -2,4 +2,8 @@ module ProjectHelper
   def current_project
     @current_project ||= Projects::Find.new(params[:project_name], current_user).run!
```

Comment:
the project and remote project may have a different name

I don't know how to handle that for the moment